src/components/playlist/playlist-big.html

```diff
@@ -0,0 +1,13 @@
+<!-- klassischer Ansatz zur Nutzung von filter -->
```

Comment:
would recommend to write comment in english

Reply:
 🙈 nooboo alarm, of course